docs/content/installation/integrations/opentracing.md

```diff
@@ -10,24 +10,26 @@ docs: "DOCS-618"
 ---
 
 
-The Ingress Controller supports [OpenTracing](https://opentracing.io/) with the third-party module [opentracing-contrib/nginx-opentracing](https://github.com/opentracing-contrib/nginx-opentracing).
+NGINX Ingress Controller supports [OpenTracing](https://opentracing.io/) with the third-party module [opentracing-contrib/nginx-opentracing](https://github.com/opentracing-contrib/nginx-opentracing).
 
-This document explains how to use OpenTracing with the Ingress Controller.
-
-**Note**: The examples below use the snippets annotations, which are disabled by default. To use snippets, set the [`enable-snippets`]({{< relref "configuration/global-configuration/command-line-arguments#cmdoption-enable-snippets" >}}) command-line argument.
+{{< note >}} The examples below use the snippets annotations, which are disabled by default. To use snippets, set the [`enable-snippets`]({{< relref "configuration/global-configuration/command-line-arguments#cmdoption-enable-snippets" >}}) command-line argument.{{< /note >}}
```

Comment:
Can we move this to the prereqs section? e.g.

```
2. Enable snippets annotations by setting the [`enable-snippets`]({{< relref "configuration/global-configuration/command-line-arguments#cmdoption-enable-snippets" >}}) command-line argument to true.
```

Reply:
resolved via commit